pkg/cmd/cli/describe/printer.go

```diff
@@ -161,6 +161,13 @@ func PrintTemplateParameters(params []templateapi.Parameter, output io.Writer) e
 	return nil
 }
 
+// Receives a resource kind and its name, returns a formatted
+// string with the "resource_kind/resource_name"
+func formatResourceName(kind, name string) string {
+	rname := kind + "/" + name
```

Comment:
Suggestion:

```go
func TestFormatResourceName(t *testing.T) {
	tests := []struct {
		kind, name string
		want       string
	}{
		{"", "", ""},
		{"", "name", "name"},
		{"kind", "", "kind/"}, // should not happen in practice
		{"kind", "name", "kind/name"},
	}
	for _, tt := range tests {
		if got := formatResourceName(tt.kind, tt.name); got != tt.want {
			t.Errorf("formatResourceName(%q, %q) = %q, want %q", tt.kind, tt.name, got, tt.want)
		}
	}
}
```

Now, if you make `formatResourceName` pass those tests, you don't need to write a lot of ifs in every `print*` function, you could always say:

```go
name := formatResouceName(opts.Kind, v.Name) // where v is t, build, bc, etc, depending on the arg.
// No need to store opts.Kind in a variable, it is ever only used above.
```

WDYT?

Would be nice to follow the same pattern in the Kubernetes code. There we're not using anything like `formatResouceName`, but repeating the ifs and concatenating the strings in every place.